tests/applepay_test.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timedelta
+
+from pytz import utc
+
+from applepay import payment
+from applepay import utils as applepay_utils
+
+import utils as test_utils
+
+
+def test_retrieve_signature_signing_time():
+    # Given a detached CMS signature in the token,
+    token = test_utils.load_json_fixture('tests/fixtures/token.json')
```

Comment:
I think with pytest.fixtures and the scope kwarg it would be possible to load this data once for the entire test file.  Since the data is not changing from test to test it may save on test runtime to only hit the filesystem it once.